lib/correios/cep/address_finder.rb

```diff
@@ -19,6 +22,21 @@ def web_service
       def parser
         @parser ||= Correios::CEP::Parser.new
       end
+
+      private
+
+      def format_response(response)
```

Comment:
@prodis I agree with you. I'll change the name of this method and send another PR.